src/test/ui/consts/const-eval/pub_const_err.rs

```diff
@@ -7,4 +7,5 @@ pub const Z: u32 = 0 - 1;
 //~^ WARN any use of this value will cause an error
 //~| WARN this was previously accepted by the compiler but is being phased out
 
-pub type Foo = [i32; 0 - 1];
+//pub type Foo = [i32; 0 - 1];
+//^ evaluation of constant value failed
```

Comment:
Please turn these into error tests (see https://rust-lang.zulipchat.com/#narrow/stream/213817-t-lang/topic/erroneous.20constants.20in.20type.20alias/near/229492342 for "lang team sign off on making this a hard error if crater is clean")